src/api/app/assets/stylesheets/webui/application/comments.scss

```diff
@@ -17,9 +17,6 @@
   border-top: none;
 }
 
-.comment-actions {
```

Comment:
grep said no. But of course it won't harm to check the ui again after removing it